src/Uno.UI/UI/Xaml/Application.cs

```diff
@@ -75,6 +86,16 @@ public ApplicationTheme RequestedTheme
 
 		public event UnhandledExceptionEventHandler UnhandledException;
 
+		public void OnSystemThemeChanged()
+		{
+			// if user overrides theme, don't apply system theme
+			if (!_themeSetExplicitly)
+			{
+				var theme = GetDefaultSystemTheme();
+				SetRequestedTheme(theme);
+			}
```

Comment:
Should `_themeSetExplicitly` be reset to `false` here?

Reply:
The `_themeSetExplicitly` variable is set to `true` only if user sets the theme in `Application.xaml`, and after it becomes `true`, it cannot become `false` anymore (the app either reacts to system theme automatically, or the developer is fully in control)